test/convert/datetime.jl

```diff
@@ -135,3 +135,8 @@ r = RObject(d)
 @test ismissing(rcopy(r)[2])
 @test rcopy(Array, r)[[1,3]] == d[[1,3]]
 @test ismissing(rcopy(Array, r)[2])
+
+# microseconds on R side #396
+@test_logs((:debug, "Precision lost in conversion to DateTime"),
+            min_level=Logging.Debug,
+            rcopy(R"as.POSIXct('2020-10-09 12:09:46.1234')"))
```

Comment:
It'd probably be worthwhile to test the result as well.
```suggestion
            rcopy(R"as.POSIXct('2020-10-09 12:09:46.1234')"))
@test rcopy(R"as.POSIXct('2020-10-09 12:09:46.1234')") == DateTime("2020-10-09T12:09:46.123")
```